keps/sig-network/1435-mixed-protocol-lb/README.md

```diff
@@ -587,12 +592,12 @@ the health of the service?**
 
 * **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
 
-  TBD
+  N/A
```

Comment:
Not N/A. I expect that we will define SLOs and SLIs that define what "normal" quality of service looks like for this feature. You write below:

> **Will enabling / using this feature result in increasing time taken by any operations covered by [existing SLIs/SLOs]?**
> The setup of more ports/listeners with different protocols may take more time, depending on how the CPI and the cloud provider API is implemented

so let's include something here about our expectations.

Reply:
Well, the functional change is that you can use this now. What if it doesn't perform as well as a standard non-mixed LB? You don't need to give me paragraphs here, just a couple sentences on what the expectations for performance for the feature are.